src/NHibernate.Test/NHSpecificTest/NH1001/Mappings.hbm.xml

```diff
@@ -3,7 +3,7 @@
 				   namespace="NHibernate.Test.NHSpecificTest.NH1001"
 				   assembly="NHibernate.Test">
 
-	<class name="Department" table="DEPARTMENTS" lazy="false">
+	<class name="Department" table="DEPARTMENTS" lazy="false" entity-name="FooDepartment">
```

Comment:
This `class` xml element needs to stay. I meant to add new xml element `class` with `entity-name`